pkg/shim/scheduler.go

```diff
@@ -145,6 +146,10 @@ func (ss *KubernetesShim) register(e *fsm.Event) {
 
 func (ss *KubernetesShim) handleSchedulerFailure(e *fsm.Event) {
 	ss.stop()
+	// testmode will be true when mock scheduler intailize
+	if configured := conf.GetSchedulerConf(); !configured.IsTestMode() {
```

Comment:
this should be 
```
if !conf.GetSchedulerConf().IsTestMode() {
  ...
}
```
conf.GetSchedulerConf() should never be nil.

Reply:
ok